src/DssSpell.sol

```diff
@@ -52,164 +49,43 @@ contract DssSpellAction is DssAction {
-
-        // WBTC-C: Decrease the Stability Fee by 2 percentage points from 11.50% to 9.50%
-        DssExecLib.setIlkStabilityFee("WBTC-C", NINE_PT_FIVE_PCT_RATE, /* doDrip = */ true);
-
-        // DSR: Decrease the Dai Savings Rate by 2 percentage points from 10.00% to 8.00%
-        DssExecLib.setDSR(EIGHT_PCT_RATE, /* doDrip = */ true);
-
-        // ---------- Delegate Compensation ----------
-        // Forum: https://forum.makerdao.com/t/april-2024-aligned-delegate-compensation/24272
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-income-and-participation-requirements
-
-        // BLUE - 41.67 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 41.67 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Cloaky - 41.67 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 41.67 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // BONAPUBLICA - 28.23 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        MKR.transfer(BONAPUBLICA, 28.23 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // TRUE NAME - 27.52 MKR - 0x612F7924c367575a0Edf21333D96b15F1B345A5d
-        MKR.transfer(TRUENAME, 27.52 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // JuliaChang - 24.20 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 24.20 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // vigilant - 13.89 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
-        MKR.transfer(VIGILANT, 13.89 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Pipkin - 13.44 MKR - 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7
-        MKR.transfer(PIPKIN, 13.44 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // JAG - 7.17 MKR - 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f
-        MKR.transfer(JAG, 7.17 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Byteron - 6.72 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 6.72 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // UPMaker - 3.58 MKR - 0xbB819DF169670DC71A16F58F55956FE642cc6BcD
-        MKR.transfer(UPMAKER, 3.58 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        //Approve RWA004-A Dao Resolution with IPFS hash QmUiYTRy4BkV681tfFe3Ksj6gdLEq27w34MqLw5LvRaBoD
+        // Note: see `dao_resolutions` variable declared above
 
         // ---------- Spark Spell ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250
+        // Forum: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/3
```

Comment:
Nit: please use Reasoning URL instead of the Authority URL by default, i.e.:

```diff
-        // Forum: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/3
+        // Forum: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327
```

Reply:
Hmm i guess then it a mistake from governance, they have the same link in both Reasoning and Authority tab, i was confused when i saw it